modin/engines/base/io/text/csv_reader.py

```diff
@@ -110,8 +109,12 @@ def _read(cls, filepath_or_buffer, **kwargs):
                     skiprows += header + 1
                 elif hasattr(header, "__iter__") and not isinstance(header, str):
                     skiprows += max(header) + 1
-                for _ in range(skiprows):
-                    f.readline()
+                cls.read_rows(
+                    f,
+                    skiprows,
```

Comment:
Shouldn't you read `nrows` first?

Reply:
no, when `nrows` is specified we counting the number of rows that we read excluding `skiprows`.

Case of using `nrows` + `skiprows` is covered by `test_from_csv_skiprows`